src/i18n/fr.json

```
@@ -321,7 +321,7 @@
           "price_info": "Astuce: ajoutez des informations sur les prix",
           "contact_info": "Astuce: ajoutez des informations de contact",
           "organizer": "Astuce: ajouter un organisateur",
-          "completed": "You're the best"
+          "completed": "Tous les points! Vous allez marquer des points avec cet événement!"
```

```suggestion
          "completed": "Tous les points ! Vous allez marquer des points avec cet événement !"
```

@brampauwelyn corrected me on this recently 😄 
https://www.traduix.com/diensten/veelgestelde-vragen/waarom-staan-er-spaties-voor-leestekens-in-mijn-franse-vertaling#:~:text=Een%20uitroepteken%2C%20een%20vraagteken%2C%20een,de%20zin%20nooit%20een%20spatie.

---

Ah, that's why Deepl added this extra space. Thought it was a bug or something so I didn't pay much attention to it.

We should fix this for the entire fr.json, the translation files for the angular app, widgets, etc 👉  https://jira.uitdatabank.be/browse/III-5292